tripal_chado/tripal_chado.install

```diff
@@ -545,3 +526,43 @@ function tripal_chado_update_10405() {
     throw new UpdateException('Could not add entity id to fields: ' . $e->getMessage());
   }
 }
+
+/**
+ * Updates missing mapping of CV terms to some chado table columns.
+ */
+function tripal_chado_update_10406() {
+  // Same as former 10402 but adding a new cvterm local:lineageex for PR#TBD
+  try {
+    $messenger = \Drupal::messenger();
+
+    // create the local:lineageex term for PR#TBD if it does not exist
+    $manager = \Drupal::service('tripal.collection_plugin_manager.idspace');
+    $idspace_obj = $manager->loadCollection('local');
+    if (!is_object($idspace_obj)) {
+      throw new UpdateException('Could not load local idspace');
+    }
+    $term = $idspace_obj->getTerm('lineageex');
+    if (!$term) {
+      $messenger->addMessage('Adding lineageex term');
+      $term = new TripalTerm([
+        'idSpace' => 'local',
+        'accession' => 'lineageex',
+        'vocabulary' => 'local',
+        'name' => 'lineageex',
+        'definition' => 'NCBI Extended Taxonomic Lineage',
+      ]);
+      $idspace_obj->saveTerm($term);
+    }
```

Comment:
I am open to suggestions as to whether there is a simpler way to add a new term in an update hook!

Reply:
I have made PR #1844 a dependency so that I can use a Chado Buddy to upsert the missing term!